--- a/.tox/venv/share/diskimage-builder/elements/bootloader/pkg-map    2017-01-10 18:24:09.318083000 +0000
+++ b/.tox/venv/share/diskimage-builder/elements/bootloader/pkg-map   2017-01-10 17:51:25.123492810 +0000
@@ -11,12 +11,14 @@
     },
     "redhat": {
       "extlinux": "syslinux-extlinux",
-      "grub-pc": "grub2-tools grub2"
+      "grub-pc": "grub2-tools grub2",
+      "grub-ieee1275": "grub2-tools grub2"
     }
   },
   "default": {
     "dkms_package": "dkms",
     "extlinux": "extlinux",
-    "grub-pc": "grub-pc"
+    "grub-pc": "grub-pc",
+    "grub-ieee1275": "grub-ieee1275"
   }
 }
