--- a/tox.ini
+++ b/tox.ini
@@ -5,7 +5,8 @@ skipsdist = True
 
 [testenv]
 usedevelop = True
-install_command = pip install -c{env:UPPER_CONSTRAINTS_FILE:https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt} {opts} {packages}
+#install_command = pip install -c{env:UPPER_CONSTRAINTS_FILE:https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt} {opts} {packages}
+install_command = pip install -c https://git.openstack.org/cgit/openstack/requirements/plain/upper-constraints.txt {opts} {packages}
 setenv =
     VIRTUAL_ENV={envdir}
 deps =
