--- a/usr/lib/python2.7/site-packages/fuel_health/heatmanager.py  2016-12-02 18:16:02.315176819 +0000
+++ b/usr/lib/python2.7/site-packages/fuel_health/heatmanager.py 2016-12-01 18:41:24.576755314 +0000
@@ -46,7 +46,7 @@
         if not self.find_micro_flavor():
             self.fail('m1.micro flavor was not created.')
 
-    def create_flavor(self, ram=256, vcpus=1, disk=2):
+    def create_flavor(self, ram=256, vcpus=1, disk=3):
         """This method creates a flavor for Heat tests."""
 
         LOG.debug('Creation of Heat tests flavor...')
